grace/evaluation/inference.py

```diff
@@ -40,45 +51,77 @@ def set_node_and_edge_probabilities(self, G: nx.Graph):
             prediction = (int(e_pred[e_idx].item()), e_probabs[e_idx].numpy())
             edge[-1][GraphAttrs.EDGE_PREDICTION] = prediction
 
-    def visualise_performance(self, G: nx.Graph):
+    def visualise_model_performance_on_test_graph(self, G: nx.Graph):
```

Comment:
why `visualise_model_performance_on_test_graph` and not `visualise_model_performance_on_graph`, will there be a different function for looking at performance within the training graph? Still should be general right?

Reply:
Very good point, committing the change now!